### ISSUE TITLE
How to implement OrthDNN on ImageNet ?

### ISSUE THREAD

**Issue by CollinGuo** [Gorilla-Lab-SCUT/OrthDNNs#1]
I can not reproduce the results of SVB on ImageNet ? Could you tell me what is the settings of weight decay, initial learning rate, learning rate decay methods, and number of epochs ?
